source/access/how_to_request_more_quota.rst

```diff
@@ -11,9 +11,32 @@ option depends on data storage policies of the site managing your VSC account,
 VO or Tier-1 project as well as on current capacity of the storage system.
 
 Before requesting more storage, please check carefully the
-:ref:`current data usage of your VSC account <quota>` and identify which
+:ref:`current data usage of your VSC account <checking disk usage>` and identify which
 file system needs a larger quota.
 
+.. _more quota inode:
+
+Requesting more inode quota
+===========================
+
+If you need more :ref:`inode quota <quota>`, you should first try to
+reduce the number of files as much as possible:
+
+- Clean up the files you no longer need.
+- Pack the remainder in a tarball, e.g. ``tar zcf myfiles.tgz file1 <...>
+  fileN``. If you need help on ``tar``, please consult the man page, *i.e.*, do:
+  ``man tar``.
+- If tarballs are not a suitable option for you, consider agregating your data
+  into large ``CSV``, ``HDF5``, or ``netCDF`` files, or using a database such as
+  ``SQLite``, ``DuckDB``, ``MariaDB``, or ``PostgreSQL``.
```

Comment:
duckdb? never heard of it before this PR; not packaged in el8 and not in EB? 
why do you bring this up here?

Reply:
it's just a suggestion, just like the other options.
duckb is optimized for analytical workloads, which is probably what most researchers do.
it seems to be getting quite popular.
it's just a python package so probably not an issue to install.
no problem to remove it if you care so much, but why are you so against this?

Reply:
it's not just a python package, it's probably not available on the clusters at this moment, from the website it seems not optimal for HPC at all: why do you mention it? did you have some recent success from users trying it?

Reply:
i don't have experience with it, nor have our users that i know if.
it's a popular sqlite alternative and is claimed to be more performant for analytical workloads as it is column-based instead of row-based like sqlite.
there is no one-size-fits-all in DBs. it may not be optimal for HPC, but neither is sqlite.

i'll remove it.

Reply:
i would also not recomend sqlite tbh ;)